### DISCUSSION TITLE
all new features should go to j4.1

### DISCUSSION THREAD

**Discussion by ghost** [joomla/joomla-cms#32527] [General]
all new features should go to j4.1

or we are in stage there new features should be added in j4.0?

**Comment by ghost** [comment]
new feature which are not needed for all create new issues, please stop proposing them to j4 but better propose them to 4.1 or later.

We want Joomla 4 to release NOW, so please stop wasting time of other people, and better test release blockers.

**Comment by chmst** [comment]
Please stop spamming this discussion and waste our time. Release leads decide what to merge where and when.
